### ISSUE TITLE
Add a small space for notes?

### ISSUE THREAD

**Issue by TheNomad11** [simjanos-dev/LinguaCafe#277]
When I use LinguaCafe as one of the  main space for learning languages wouldn*t it be useful to have a space for notes with links to useful websites, things to do and other stuff? Noting fancy, just a space one note might be enough? What do you think?

**Comment by simjanos-dev** [comment]
Do you mean just one textarea? I had the same idea, but wasn't sure if anyone would find it useful. I was planning on adding a lot of stuff to it though.

**Comment by TheNomad11** [comment]
yeah, exactly my thought, just a simple text area (with clickable links). what were you planning to add!?

**Comment by simjanos-dev** [comment]
I wanted to add an actual notebook. 

- Multiple pages
- Drawing with preassure sensitive pencils
- Todo lists
- Linkining words, achivements and stuff
- Adding images

Havent thought it through, but basically a language learning notebook/journal. 

Last time I had it in my mind was before release, then got a lot of unexpected tasks and kind of forgot about it. 

I think I could just start with a simple one the homepage though, that is quick to implement. I can render it with markdown, so you can add links and a lot of other stuff to it easily.

**Comment by TheNomad11** [comment]
Cool! Yeah, agree, that would be a good start!
